bin/start-shuffle-server.sh

```diff
@@ -30,7 +30,8 @@ SHUFFLE_SERVER_CONF_FILE="${RSS_CONF_DIR}/server.conf"
 JAR_DIR="${RSS_HOME}/jars"
 LOG_CONF_FILE="${RSS_CONF_DIR}/log4j2.xml"
 LOG_PATH="${RSS_LOG_DIR}/shuffle_server.log"
-AUDIT_LOG_PATH="${RSS_LOG_DIR}/audit.log"
+SHUFFLE_SERVER_STORAGE_AUDIT_LOG_PATH="${RSS_LOG_DIR}/shuffle_server_storage_audit.log"
```

Comment:
Maybe `storage_audit.log` and `rpc_audit.log` will be enough? WDYT? Are their names too long currently?

@maobaolong

Reply:
@rickyma I suppose to introduce coordinator RPC Auditlog next

Reply:
They are different startup scripts. I think it's ok to have the same name?

Reply:
Discussed with @rickyma , The scripts of `uniffle` could do some refactor to abstract configure purpose vars into `rss-env.sh`, then put `rss-env.sh` into `conf` folder, eventually, we can maintain scripts by git repository and maintain conf by conf servers.

Reply:
@rickyma This discussion have been tracked by #1959 and resolved by #1957